modules/ROOT/pages/deploying-to-cloudhub.adoc

```diff
@@ -108,17 +106,19 @@ With the CloudHub dedicated load balancer, you can then optionally implement xre
 
 == Deployment Target
 
-If you already have any servers registered to the Anypoint Platform, you will see a *Deployment Target* field that lets you specify where you want to deploy your application:
+If you already have servers registered to the Anypoint Platform, you see a *Deployment Target* field that lets you specify where you want to deploy your application:
```

Comment:
is this easier to read? "...you see a Deployment Target field. Use it to specify where you want...."